.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,5 +13,9 @@
 
 ## Checklist
 <!-- Put an `x` in the boxes. All tasks must be completed and boxes checked before merging. -->
+- [ ] 🤖 This change is covered by unit tests (if applicable).
+- [ ] 🤹 Manual testing has been performed (if necessary).
+- [ ] 🛡️ Security impacts have been considered (if relevant).
+- [ ] 📖 Documentation updates are complete (if required).
```

Comment:
Could relax this to "Documentation updates are complete or captured in separate issue (if relevant)." It's sometimes easier to write a lot of docs updates in one go rather than binding them to a single PR.

Reply:
We probably need to discuss the concept.
I was thinking along the lines of assessments. All questions need to answered yes and if not there needs to be a reason for deviation, like e.g. adding a link to an issue that captures this (and other) updates to the documentation.

Reply:
OK, needs an update of the description above the list then to explain this.